Content.Server/Physics/Controllers/ConveyorController.cs

```diff
@@ -13,14 +13,13 @@ namespace Content.Server.Physics.Controllers
 {
     internal sealed class ConveyorController : VirtualController
     {
-        private ConveyorSystem _conveyor = default!;
-
-        public override List<Type> UpdatesAfter => new() {typeof(MoverController)};
+        [Dependency] private readonly ConveyorSystem _conveyor = default!;
 
         public override void Initialize()
         {
             base.Initialize();
-            _conveyor = EntitySystem.Get<ConveyorSystem>();
+
+            UpdatesAfter.Add(typeof(MoverController));
```

Comment:
adding the updatesafter after the basecall which uses it. :sus:

move it infront of the basecall

Reply:
true, good catch!

Reply:
fixed now